tests/e2e/env/config/default/default.json

```diff
@@ -1,5 +1,6 @@
 {
   "url": "http://localhost:8084/",
+  "appName": "woocommerce_e2e",
```

Comment:
This file also needs to be reverted. The `env` version of the file should not have `appName` but the local config should have it.